policies/accessing-sensitive-information-policy.md

```diff
@@ -0,0 +1,113 @@
+# Accessing Sensitive Information Policy

```

Comment:
This shouldn't be here.  I suspect you branched from a branch rather than main/master.